### PR TITLE
status: Fetch only valid status updates

### PR DESCRIPTION
**Pull request by harshithpabbati**
- Fetches mail only if it's sent to ML
- Update Telegram Report

Output:
Screenshot 2020-01-15 at 12 06 53 AM

### CODE REVIEW

File: status/logger.py [hunk 1]
```diff
@@ -14,32 +14,46 @@ def log(data, members, thread_id):
...
+                            obj.didNotSend.remove(user)
+                    obj.save()
+            else:
                 obj, created = DailyLog.objects.get_or_create(
                     date=logDate,
                     thread=thread,
                 )
+
                 if created:
                     obj.members.add(*members)
```
Comment (vibhoothi) [inline status/logger.py:52]: Shouldn't we have object for invalid updates here too? 
like 
 obj.invalidUpdates.add(*members)

is it required, if not why?

File: status/StatusUpdateReporter.py [hunk 2]
```diff
@@ -156,12 +167,14 @@ def generateDailyReport(self):
 
             totalMembers = log.members.count()
             didNotSendCount = log.didNotSend.count()
-            sendCount = totalMembers - didNotSendCount
+            invalidUpdatesCount = log.invalidUpdates.count()
+            sendCount = totalMembers - didNotSendCount - invalidUpdatesCount
```
Comment (vibhoothi) [inline status/StatusUpdateReporter.py]:
```suggestion
            sendCount = totalMembers - (didNotSendCount + invalidUpdatesCount)
```

Isn't this convention better

**Review by vibhoothi — COMMENTED** [review]
Hi,

Thank you for this patchset,

I have small nits apart from that It LGTM.

File: status/logger.py (hunk 1 above)
Comment (harshithpabbati) [reply status/logger.py:52]: No, We don't need to add everyone. We can just add the users who have the invalid status update.

Comment (vibhoothi) [reply status/logger.py:52]: ok

**Review by vibhoothi — APPROVED** [review]
LGTM

**Pull request merged**